spec/javascripts/govuk-component/govspeak-enhance-youtube-video-links-spec.js

```diff
@@ -0,0 +1,67 @@
+describe('Links to YouTube in govspeak content', function () {
+  "use strict";
+
+  var enhanceYoutubeVideoLinksFixture, $enhanceYoutubeVideoLinksHTML, enhanceYoutubeVideoLinks;
+
+  var turnOffEnhanceYoutubeVideoLinksFixture, $turnOffEnhanceYoutubeVideoLinksHTML;
+
+  describe('- A YouTube link to be converted to an embedded player', function () {
+
+    beforeEach(function() {
+
+      enhanceYoutubeVideoLinksFixture = '<div class="govuk-govspeak">'+
+                      '<a href="https://youtu.be/OzjogCFO4Zo">'+
+                        'This link to YouTube'+
+                      '</a>'+
+                    '</div>';
+
+      $enhanceYoutubeVideoLinksHTML = $(enhanceYoutubeVideoLinksFixture);
```

Comment:
You could simplify this by going straight to `$enhanceYoutubeVideoLinksHTML` without the intermediate fixture:

eg
```
$enhanceYoutubeVideoLinksHTML = $('<div class="govuk-govspeak">'+
  '<a href="https://youtu.be/OzjogCFO4Zo">'+
    'This link to YouTube'+
  '</a>'+
'</div>');
```